CONTRIBUTING.md

```diff
@@ -5,7 +5,32 @@
 If you are interested in contributing to pyOpenSci, please read about
 how we work in [**pyOpenSci's organization-wide contributing guide**](https://www.pyopensci.org/handbook/CONTRIBUTING.html).
 
-## Repositories
+## Updating blog posts
```

Comment:
This is a new feature. We can automatically update the last_updated key on a post if that key exists. It then ensures that page updates at the bottom of the page is the actual date it was updated rather than the date the page was created (taken from the filename) vs commit history. 
<img width="388" alt="Screenshot 2024-08-28 at 6 53 12 PM" src="https://github.com/user-attachments/assets/6a919209-74b2-469d-8202-287e8abe40dc">